packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -124,6 +124,7 @@ describe("Hotkeys", () => {
             // wait for the dialog to animate in
             setTimeout(() => {
                 expect(document.querySelector(".pt-hotkey-column")).to.exist;
+                expect(document.querySelector(".pt-overlay-open").classList.contains("pt-overlay-inline")).to.be.false;
```

Comment:
I'm ok with merging this for the PR, but this is a tautological test that doesn't really help to prevent bugs (it matches exactly what you wrote in the component markup / props)